README.md

```diff
@@ -40,22 +40,20 @@ RisingLight developers are active in a variety of places:
 
 ### Slack Channel
 
-The RisingLight project is currently part of the RisingWave community. You may join RisingWave community with the [invitation link](https://join.slack.com/t/risingwave-community/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw). If the Slack link expires (which normally should not happen), please create an issue :)
+The RisingLight Slack channel is currently hosted in the RisingWave community. You may step in the RisingWave community with the [invitation link](https://join.slack.com/t/risingwave-community/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw), and then join the following channels:
```

Comment:
> If the Slack link expires (which normally should not happen), please create an issue :)

I would prefer retaining this part. I've clicked into a lot of expired Slack links in other communities. It seems that the link would expire frequently without a reason.